docs/apache-airflow/security/access-control.rst

```diff
@@ -37,101 +37,113 @@ regarding its security model.
 
 Default Roles
 '''''''''''''
-Airflow ships with a set of roles by default: Admin, User, Op, Viewer, and Public.
-Only ``Admin`` users could configure/alter the permissions for other roles. But it is not recommended
-that ``Admin`` users alter these default roles in any way by removing
-or adding permissions to these roles.
+
+Airflow uses roles to manage all permissions across your environment. Each role has permissions which provide varying levels of access to Airflow's key resources (DAGs, Connections, etc).
+
+An environment's roles can be found under the **Security** tab:
+
+.. image:: /img/list-roles.png
+
+Airflow ships with a set of roles by default:
 
 Admin
 ^^^^^
 ``Admin`` users have all possible permissions, including granting or revoking permissions from
 other users.
 
+Only ``Admin`` users can configure/alter the permissions for other roles. While they can reconfigure Airflow's other default rules, it's not recommended.
+The best practice is to instead create a new role with the desired permissions.
+
 Public
 ^^^^^^
 ``Public`` users (anonymous) don't have any permissions.
```

Comment:
I wanted to add more info here, but I personally don't know much about the Public role and how it's used in Airflow. Any suggestions for additions here?

Reply:
Public is the role you get when you aren't logged in -- if you gave some permissions to this you could let all people with access to your Airflow webserver view things.